### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by Guptacos**
Fix minor style things in many files to pass the linter

Change how the linter finds files, making it more efficient and removing files we don't want to lint

### CODE REVIEW

**Comment by Guptacos** [comment]
Not ready for review

**Comment by spenceryu** [comment]
> Not ready for review

Github has an option to set a PR as a draft.

**Review by spenceryu — DISMISSED** [review]
lgtm

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Get all python files tracked by git
-ALL_LINT_FILES := $(shell git ls-files | grep '**.py' | xargs)
-DIFF_LINT_FILES := $(shell git diff origin/master --name-only | grep '**.py' | xargs)
+ALL_LINT_FILES := $(shell python lint.py all)
+DIFF_LINT_FILES := $(shell python lint.py diff)
```
Comment (rmannava) [inline Makefile]: python3?

**Pull request merged**